docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -193,31 +255,216 @@ You should receive a response indicating there were no errors.
+
+[discrete]
+[[getting-started-dynamic-mapping]]
+==== Use dynamic mapping
+
+When using dynamic mapping, {es} automatically creates mappings for new fields by default.
+The documents we've added so far have used dynamic mapping, because we didn't specify a mapping when creating the index.
+
+Add a new document to the `books` index with a field that doesn't exist, to see how dynamic mapping works.
+
+[source,console]
+----
+POST /books/_doc
+{
+  "name": "The Great Gatsby",
+  "author": "F. Scott Fitzgerald",
+  "release_date": "1925-04-10",
+  "page_count": 180,
+  "new_field": "I'm a new field added dynamically" <1>
+}
+----
+// TEST[continued]
+<1> This field is dynamically added to the document.
+View the mapping for the `books` index. The new field `new_field` has been added to the mapping with a `text` data type.
+
+[source,console]
+----
+GET /books/_mapping
+----
+// TEST[continued]
+
+.Expand to see example response
+[%collapsible]
+===============
+[source,console-result]
+----
+{
+  "books": {
+    "mappings": {
+      "properties": {
+        "author": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
+            }
+          }
+        },
+        "name": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
+            }
+          }
+        },
+        "new_field": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
+            }
+          }
+        },
+        "page_count": {
+          "type": "long"
+        },
+        "release_date": {
+          "type": "date"
+        }
+      }
+    }
+  }
+}
+----
+// TEST[continued]
+===============
+
+[discrete]
+[[getting-started-explicit-mapping]]
+==== Define explicit mapping
+
+Create an index named `my-explicit-mappings-books` with explicit mappings:
+
+[source,console]
+----
+PUT /my-explicit-mappings-books
+{
+  "mappings": {
+    "dynamic": false,  <1>
+    "properties": {  <2>
+      "name": { "type": "text" },  <3>
+      "author": { "type": "text" },
+      "release_date": { "type": "date", "format": "yyyy-MM-dd" },  <4>
+      "page_count": { "type": "integer" }  <5>
+    }
+  }
+}
+----
+// TEST[continued]
+<1> Disables dynamic mapping for the index. Documents containing fields not defined in the mapping will be rejected.
+<2> The `properties` object defines the fields and their data types for documents in this index.
+<3> Defines the "name" field as type "text", suitable for full-text search on book titles.
+<4> Specifies the "release_date" field as a "date" type with the specific format `yyyy-MM-dd`.
+<5> Sets the "page_count" field as an "integer" type for storing the number of pages.
```

Comment:
again possibly overkill. what I was looking for was something like 

> Pass each field's properties as a JSON object. This object should contain the field type and any additional [mapping parameters](https://elasticsearch_bk_113558.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/mapping-params.html).

would skip annotations 3-5 in favor of this.

Reply:
> Pass each field's properties as a JSON object. This object should contain the field type and any additional [mapping parameters](https://elasticsearch_bk_113558.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/mapping-params.html).


Ooooh nice